components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaFetchRecords.java

```diff
@@ -499,4 +510,19 @@ long getCurrentRecoveryInterval() {
     public BridgeExceptionHandlerToErrorHandler getBridge() {
         return bridge;
     }
+
+    /*
+     * This is for manually pausing the consumer
+     */
+    public void pause() {
+        consumer.pause(consumer.assignment());
+    }
+
+    /*
+     * This is for manually resuming the consumer (not to be confused w/ the Resume API). This is
+     * mostly used for directly calling pause from Java code or via JMX
```

Comment:
should this javadoc not be added to the pause method above

Reply:
Yeap. Fixing it, thanks!